docs/installation.md

```diff
@@ -64,40 +67,63 @@ You may have to set your specific configurations if it is deployed into a produc
+| `imagePullSecrets`                        | The list of image pull secrets for kruise image              | `[]`                       |
+
+
+#### manager parameters
+| Parameter                                 | Description                                                  | Default                       |
+| ----------------------------------------- | ------------------------------------------------------------ | ----------------------------- |
 | `manager.log.level`                       | Log level that kruise-manager printed                        | `4`                           |
 | `manager.replicas`                        | Replicas of kruise-controller-manager deployment             | `2`                           |
 | `manager.image.repository`                | Repository for kruise-manager image                          | `openkruise/kruise-manager`   |
-| `manager.image.tag`                       | Tag for kruise-manager image                                 | `v1.6.3`                      |
+| `manager.image.tag`                       | Tag for kruise-manager image                                 | `v1.7.0`                      |
 | `manager.resources.limits.cpu`            | CPU resource limit of kruise-manager container               | `200m`                        |
 | `manager.resources.limits.memory`         | Memory resource limit of kruise-manager container            | `512Mi`                       |
 | `manager.resources.requests.cpu`          | CPU resource request of kruise-manager container             | `100m`                        |
 | `manager.resources.requests.memory`       | Memory resource request of kruise-manager container          | `256Mi`                       |
 | `manager.metrics.port`                    | Port of metrics served                                       | `8080`                        |
 | `manager.webhook.port`                    | Port of webhook served                                       | `9443`                        |
+| `manager.pprofAddr`                       | Address of pprof served                                      | `localhost:8090`              |
 | `manager.nodeAffinity`                    | Node affinity policy for kruise-manager pod                  | `{}`                          |
 | `manager.nodeSelector`                    | Node labels for kruise-manager pod                           | `{}`                          |
 | `manager.tolerations`                     | Tolerations for kruise-manager pod                           | `[]`                          |
+| `manager.resyncPeriod`                    | Resync period of informer kruise-manager, defaults no resync | `0`                           |
+| `manager.hostNetwork`                     | Whether kruise-manager pod should run with hostnetwork       | `false`                       |
+| `manager.loggingFormat`                   | Whether structured logging, for example `json`    | ` `                       |
```

Comment:
plz describe the valid value for loggingFormat. and since the value is not boolean, consider change the description from  `Whether structured logging` to `logging format, valid formats includes ``(plain text), `json`.